workbench-core/accounts/src/services/hostingAccountService.ts

```diff
@@ -41,7 +42,18 @@ export default class HostingAccountService {
    *
    * @returns account record in DDB
    */
-  public async update(accountMetadata: UpdateAccountMetadata): Promise<{ [key: string]: string }> {
-    return await new HostingAccountLifecycleService().updateAccount(accountMetadata);
+  public async update(accountMetadata: UpdateAccountData): Promise<Record<string, string>> {
+    return await this._lifecycleService().updateAccount(accountMetadata);
+  }
+
+  private _lifecycleService(): HostingAccountLifecycleService {
```

Comment:
Why do we recreate lifecycle service vs create it once?

Reply:
because i'm a 🤡 who did poorly executed refactor. fixing this now lol